src/makeInfinityAdder.js

```diff
@@ -4,7 +4,24 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (number) => {
+    if (number !== undefined) {
+      sum += number;
+
+      return adder;
+    }
+
+    if (number === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+  };
```

Comment:
Let's write like that
```suggestion
    if (isNan(number) {
    	 const result = sum;

      sum = 0;

      return result;
    }
      sum += number;

      return adder;

  };
```

Reply:
ok, and add ')' for 'if'... however, now tests is broken